.github/workflows/codespell.yml

```diff
@@ -0,0 +1,21 @@
+name: codespell
+on: [pull_request, workflow_dispatch]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    permissions: {}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Install codespell
+        run: |
+          python3 -m pip install --break-system-packages codespell
+
+      - name: Run codespell
+        timeout-minutes: 5
+        run: |
+          codespell src
```

Comment:
FYI, it is worth to add [the pyproject.toml section](https://github.com/cockpit-project/cockpit-podman/blob/main/pyproject.toml#L80) so when running it locally it doesn't complain about node_modules.

Reply:
ah true, I should have checked the[ PR that adds this](https://github.com/cockpit-project/cockpit-podman/pull/2151/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711) for other projects